test/unittests/mongoBackend/mongoContextProvidersQuery_test.cpp

```diff
@@ -594,7 +596,7 @@ TEST(mongoContextProvidersQueryRequest, noPatternAttrsAll)
 
   EXPECT_EQ(SccOk, RES_CER_STATUS(0).code);
   EXPECT_EQ("OK", RES_CER_STATUS(0).reasonPhrase);
-  EXPECT_EQ(0, RES_CER_STATUS(0).details.size());
+  EXPECT_EQ("", RES_CER_STATUS(0).details);
```

Comment:
Not actually related with this PR, but I have take the opportunity for fix all them.